### ISSUE TITLE
Intelligent handling of breaks

### ISSUE THREAD

**Issue by meglio** [scossar/discourse-grid#2]
Can you make it so line breaks around those `div` tags are removed before processing with markdown? This way we will use the grid with the more user-friendly markdown version which converts  line breaks to `<br/>`

**Comment by scossar** [comment]
Yes that might be possible, but it's not something I have time to do now. Pull requests are welcome.
